Adding storage gloss terms (#335)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc

// OCP: General; kept as is
// Fuse: Added "In Red Hat Fuse," and removed "In Camel"
// Combined entries; moved to "with caution" since the Fuse one is with caution

[[encrypted-disk]]
==== image:images/yes.png[yes] encrypted disk (noun)

> *Description*: A disk containing data that has been encoded for the purpose of ensuring data security.

Reviewer: ```suggestion
*Description*: An _encrypted disk_ contains data that has been encoded for the purpose of ensuring data security.
```

*See also*:

[[ephemeral-storage]]
==== image:images/yes.png[yes] ephemeral storage (noun)

> *Description*: A temporary storage location that only exists while an individual pod exists, and cannot be shared by multiple pods.

Reviewer: ```suggestion
*Description*: _Ephemeral storage_ is a temporary storage location that only exists while an individual pod exists and that cannot be shared by multiple pods.
```

*Incorrect forms*: Extranet

*See also*:

[[external-mode]]
==== image:images/caution.png[with caution] external mode (noun)

> *Description*: Using externally hosted storage services, for example, a deployment of Red Hat OpenShift Container Storage that uses externally hosted Red Hat Ceph Storage (RHCS) clusters to provide storage. This OpenShift Container Storage deployment type is supported for bare metal and user-provisioned VMware environments.

Reviewer: ```suggestion
*Description*: In the context of using externally hosted storage services, for example, _external mode_ refers to a deployment of Red Hat OpenShift Container Storage that uses externally hosted Red Hat Ceph Storage (RHCS) clusters to provide storage. This OpenShift Container Storage deployment type is supported for bare metal and user-provisioned VMware environments.
```
